src/tests/Interop/MarshalAPI/IUnknown/IUnknownTest.cs

```diff
@@ -29,22 +29,12 @@ public override string ToString()
 
 public class IUnknownMarshalingTest
 {
-    [DllImport(@"IUnknownNative", CallingConvention = CallingConvention.Cdecl)]
-    private static extern bool Marshal_IUnknown([In]IntPtr ptr);
```

Comment:
Could this test just go in libraries now that it just tests the public APIs and has no native dependencies?

Reply:
They could, but I like having these basic ones in the runtime test suite as opposed to the more extensive ones in libraries.